lib/cdo/contact_rollups.rb

```diff
@@ -25,32 +26,15 @@ class ContactRollups
     query_timeout: MAX_EXECUTION_TIME_SEC
   )
 
-  # Connection to read from Pegasus reporting database.
-  PEGASUS_REPORTING_DB_READER = sequel_connect(
-    CDO.pegasus_reporting_db_reader,
-    CDO.pegasus_reporting_db_reader,
-    query_timeout: MAX_EXECUTION_TIME_SEC
-  )
-
-  # Connection to write to Pegasus reporting database.
-  PEGASUS_REPORTING_DB_WRITER = sequel_connect(
-    CDO.pegasus_reporting_db_writer,
-    CDO.pegasus_reporting_db_writer,
-    query_timeout: MAX_EXECUTION_TIME_SEC
-  )
-
-  # Connection to read from Dashboard reporting database.
-  DASHBOARD_REPORTING_DB_READER = sequel_connect(
-    CDO.dashboard_reporting_db_reader,
-    CDO.dashboard_reporting_db_reader,
-    query_timeout: MAX_EXECUTION_TIME_SEC
-  )
+  DATABASE_CLUSTER_CLONE_ID = "#{CDO.db_cluster_id}-temporary-clone"
```

Comment:
Can you share this variable with `build_contact_rollups` so it's not defined in two places?

Reply:
Done!